addons/kafka/templates/componentdef-combine.yaml

```diff
@@ -0,0 +1,203 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentDefinition
+metadata:
+  name: kafka-combine-server
+  labels:
+    {{- include "kafka.labels" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  provider: kubeblocks
+  description: |-
+    Kafka servers that act as both brokers and controllers are referred to as "combined" servers. Combined servers
+    are simpler to operate for small use cases like a development environment. Combined mode is not recommended in critical
+    deployment environments.
+  serviceKind: kafka
+  serviceVersion: {{ default .Chart.AppVersion .Values.clusterVersionOverride }}
+  services:
+  - name: broker
+    serviceName: broker
```

Comment:
combine server’s service name also called "broker"?

Reply:
yes, the combine contains the broker and controller process